app/mailers/cohort_mailer.rb

```diff
@@ -0,0 +1,34 @@
+class CohortMailer < ApplicationMailer
+  def shortlist_start_notification(user, cohort)
+    @user = user
+    @cohort = cohort
+    mail(to: @user.email, subject: 'The Shortlist Period Has Started!')
+  end
+
+  def unmatched_notification(user, cohort)
+    @user = user
+    @cohort = cohort
+    mail(to: @user.email, subject: 'Please Log In to Select a Mentor')
+  end
+  
+  def matching_complete_notification(admin, cohort)
+    @admin = admin
+    @cohort = cohort
+    mail(to: @admin.email, subject: 'Cohort Matching Complete')
+  end
+
+  def two_week_warning(admin_email, cohort)
+    @cohort = cohort
+    mail(to: admin_email, subject: "Two-Week Warning for #{@cohort.cohort_name}")
```

Comment:
If this is for the cohort ending say something like: "Two Weeks until #{@cohort.cohort_name} Ends!"
Just so they know what the warning is for.